src/scheduled-tasks/background/index.js

```diff
@@ -1,23 +0,0 @@
-import map from 'lodash/fp/map'
-
-import db from 'src/pouchdb'
-import { CronJob } from 'cron'
```

Comment:
Another package that can be removed (may be useful for scheduled tasks later, but no need right now)